_pages/about/index.html

```diff
@@ -4,31 +4,60 @@
 is_about: true
 ---
 
+<img src="/img/people/hero.jpg" />
```

Comment:
Might be worth wrapping this image, or giving it margin: auto

On large desktop screens it sits on the left and looks a bit weird
![Screenshot 2020-01-14 at 09 20 16](https://user-images.githubusercontent.com/25913173/72330847-1d7b0680-36af-11ea-84bb-df8b9c22cc9f.png)

Reply:
Have allowed it to fill large screens. 😊